pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -304,8 +304,9 @@ type MachineConfigPoolSpec struct {
 	// Label selector for Machines.
 	MachineSelector *metav1.LabelSelector `json:"machineSelector,omitempty"`
 
-	// If true, changes to this machine pool should be stopped.
-	// This includes generating new desiredMachineConfig and update of machines.
+	// If true, changes to this machine pool should be stopped,
+	// including generating new desiredMachineConfig and update of machines.
+	// This flag is intended for administrators to change.
```

Comment:
wait, IMO all the fields in spec are for edit. not just this...

Reply:
Hm...I thought I saw something else in the operator changing the spec, but trying now I can't find anything.

Dropped that patch out and updated :arrow_down: